### PR TITLE
ELEMENTS-934: Fix selectivity elements

### PR DESCRIPTION
**Pull request by akervern**

### CODE REVIEW

**Comment by nelsonsilva** [comment]
I'd rather have `export const escapeHTML = (markup) => { ... }` in `nuxeo-selectivity.js` or some `utils.js` and then you can just import it where needed `import { escapeHTML } from './nuxeo-selectivity.js';` instead of crossing all the shadow roots to find `nuxeo-selectivity`just to find a helper function that does not need to be an instance method.

File: ui/widgets/nuxeo-directory-suggestion.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ import '@nuxeo/nuxeo-elements/nuxeo-element.js';
 import { FormatBehavior } from '../nuxeo-format-behavior.js';
 import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
 import './nuxeo-selectivity.js';
+import { escapeHTML } from './nuxeo-selectivity.js';
```
Comment (guirenard) [inline ui/widgets/nuxeo-directory-suggestion.js]: This causes a lint issue. see the PR check https://qa.nuxeo.org/jenkins/job/Other/job/nuxeo-elements-pr-check/243/console

File: ui/widgets/nuxeo-document-suggestion.js [hunk 2]
```diff
@@ -22,6 +22,7 @@ import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
 import { IronFormElementBehavior } from '@polymer/iron-form-element-behavior/iron-form-element-behavior.js';
 import { RoutingBehavior } from '../nuxeo-routing-behavior.js';
 import './nuxeo-selectivity.js';
+import { escapeHTML } from './nuxeo-selectivity.js';
```
Comment (guirenard) [inline ui/widgets/nuxeo-document-suggestion.js]: same same

File: ui/widgets/nuxeo-user-suggestion.js [hunk 3]
```diff
@@ -21,6 +21,7 @@ import { IronValidatableBehavior } from '@polymer/iron-validatable-behavior/iron
 import '@nuxeo/nuxeo-elements/nuxeo-element.js';
 import '@polymer/paper-card/paper-card.js';
 import './nuxeo-selectivity.js';
+import { escapeHTML } from './nuxeo-selectivity.js';
```
Comment (guirenard) [inline ui/widgets/nuxeo-user-suggestion.js]: same same

**Comment by guirenard** [comment]
:+1:

**Pull request merged**